src/shared/components/ResourceEditor/editorUtils.spec.tsx

```diff
@@ -87,89 +108,67 @@ describe('resolveLinkInEditor', () => {
         </QueryClientProvider>
       </Provider>
     );
+    component = render(TestApp);
+    rerender = component.rerender;
   });
-  beforeEach(async () => {
-    await act(async () => {
-      await render(TestApp);
-    });
-  });
+
   afterEach(async () => {
     cleanup();
   });
   afterAll(() => {
     server.resetHandlers();
     server.close();
   });
-  // case-0: the url is not valid
+  // case-resource: can not be tested since codemirror issue and redirection
+  // case-error: link can not be resolved by the project resolver nor the search api and it's not external
   it('should return null if the url is not valid', async () => {
```

Comment:
Small thing: Maybe the `it` statement should be updated because `null` is not returned any more.